src/TRoomDB.cpp

```diff
@@ -47,8 +47,8 @@ TRoom* TRoomDB::getRoom(int id)
     if (id < 0) {
         return nullptr;
     }
-    auto i = rooms.find(id);
-    if (i != rooms.end() && i.key() == id) {
+    const auto i = rooms.constFind(id);
```

Comment:
Isn't the `const` implicit in using the `auto` keyword to hold the result of a `constFind(...)` call? :confused:

Reply:
Hm... yes, you're right